system/jcstress/playlist.xml

```diff
@@ -489,6 +489,13 @@
 		<testCaseName>jcstress_VolatileIRIWTest</testCaseName>
 		<command>$(JAVA_COMMAND) $(JVM_OPTIONS) -jar $(Q)$(LIB_DIR)$(D)jcstress-tests-all-20220908.jar$(Q) $(APPLICATION_OPTIONS) -t VolatileIRIWTest; \
 			$(TEST_STATUS)</command>
+		<disables>
+			<disable>
+				<comment>runtimes/backlog/988</comment>
+				<impl>openj9</impl>
+				<version>8</version>
```

Comment:
I noticed this test failed on JDK17. Let's set `8+`.

Reply:
Updated to 8+. Thanks.